contributors/devel/sig-node/e2e-node-tests.md

```diff
@@ -244,13 +257,15 @@ make test_e2e_node TEST_ARGS="--cgroups-per-qos=true"
 # Notes on tests run by the Kubernetes project during pre-, post- submit.
 
 The node e2e tests are run by the PR builder for each Pull Request and the results published at
-the bottom of the comments section.  To re-run just the node e2e tests from the PR builder add the comment
-`@k8s-bot node e2e test this issue: #<Flake-Issue-Number or IGNORE>` and **include a link to the test
+the bottom of the comments section. To re-run just the node e2e tests from the PR builder add the comment
+`/retest pull-kubernetes-node-e2e` and **include a link to the test
```

Comment:
One nit: `/retest` is an entirely separate command (ref: https://prow.k8s.io/command-help#retest)
```suggestion
`/test pull-kubernetes-node-e2e` and **include a link to the test
```

Reply:
Another thought might be comment an example of linking to a flake; putting info on the same line won't work
```
/test pull-kubernetes-node-e2e
flake due to #12345
```